packages/core/.github/workflows/main.yaml

```diff
@@ -0,0 +1,17 @@
+name: ci
+
+concurrency:
+  group: $-$
+  cancel-in-progress: true
+
+on:
```

Comment:
Is this in the correct folder?

Reply:
Forgot to move it. But since we already have a workflow. Should we delete this? Or keep it under a different name?

Reply:
We can delete

Reply:
I deleted it. Wanted to get it merged today 😊